engine/src/main/java/org/hibernate/validator/internal/metadata/aggregated/BeanMetaDataImpl.java

```diff
@@ -232,13 +241,7 @@ public BeanMetaDataImpl(Class<T> beanClass,
 
 	@Override
 	public boolean hasConstraints() {
-		if ( beanDescriptor.isBeanConstrained()
-				|| !beanDescriptor.getConstrainedConstructors().isEmpty()
-				|| !beanDescriptor.getConstrainedMethods( MethodType.NON_GETTER, MethodType.GETTER ).isEmpty() ) {
-			return true;
-		}
-
-		return false;
+		return hasConstraints;
```

Comment:
This one would have been better in a separate commit and JIRA I think. We can do with it though, I think.